doc/user/bgp.rst

```diff
@@ -1981,17 +1981,27 @@ Other BGP Commands
 Displaying BGP Information
 ==========================
 
+Below specific 4 command displays either main BGP IPv4 routing table, or BGP
+IPv6 routing table, depending of the presence or not of ip keyword.
+Actually, :cliref:`show ip bgp` command was used on older `Quagga` routing
```

Comment:
`:clicmd:`, not `:cliref:`.
http://docs.frrouting.org/projects/dev-guide/en/latest/workflow.html#cli-commands